mock/mock.go

```diff
@@ -209,6 +209,12 @@ func (m *Mock) On(methodName string, arguments ...interface{}) *Call {
 	return c
 }
 
+// IsCallExpected checks if a method call is expected.
+func (m *Mock) IsCallExpected(method string, arguments ...interface{}) bool {
+	found, _ := m.findExpectedCall(method, arguments...)
+	return found >= 0
+}
```

Comment:
I'm confused. Assume that the setup is:
`testEnv.On("downloadFile", "test-file-id").Return(FileInfo{Path:"path_to_downloaded_file"}, nil)
`
We would want the hasMock to be true only if the methodName is "downloadFile" and the argument is "test-file-id". With the solution you suggested, the hasMock will be true for methodName="downloadFile" with any argument.